src/lib/chain/commitment.ts

```diff
@@ -457,10 +462,11 @@ export async function collateralWithdraw(flags: CCFlags) {
     const commitmentInfo = await capacity.getCommitment(commitmentId);
     const unitIds = await market.getComputeUnitIds(commitmentInfo.peerId);
 
-    await signBatch([
-      populateTx(capacity.removeCUFromCC, commitmentId, [...unitIds]),
-      populateTx(capacity.finishCommitment, commitmentId),
-    ]);
+    for (const unitIdsBatch of chunk([...unitIds], flags[MAX_CUS_FLAG_NAME])) {
+      await sign(capacity.removeCUFromCC, commitmentId, [...unitIdsBatch]);
+    }
+
+    await sign(capacity.finishCommitment, commitmentId);
```

Comment:
Maybe if `unitIds.length < MAX_CUS`, then use `signBatch`? So that it's a single tx for the low amount of CUs.

And then catch the error, and print something like `"If you observe a problem with gas usage, try to pass a lower number to --max-cus"`